package.json

```diff
@@ -10,6 +10,7 @@
     "plugins/*"
   ],
   "nohoist": [
+    "website/landing/**",
```

Comment:
Curious to understand what made you add this

Reply:
@danilowoz trial + error to get the package working, realized later that it isn't necessary.